ocs_ci/utility/azure_utils.py

```diff
@@ -22,31 +22,45 @@
 # default location of files with necessary azure cluster details
 SERVICE_PRINCIPAL_FILEPATH = os.path.expanduser("~/.azure/osServicePrincipal.json")
 TERRRAFORM_FILENAME = "terraform.azure.auto.tfvars.json"
+OLD_TERRRAFORM_FILENAME = "terraform.azure.auto.tfvars.json"
 
 
-def load_cluster_resource_group(cluster_path, terraform_filename=TERRRAFORM_FILENAME):
+def load_cluster_resource_group(terraform_filename=None):
```

Comment:
We have discussed that we will delete this parameter as well. It's anyway not used now anyhow.